spec/migration_spec.cr

```diff
@@ -26,4 +41,17 @@ describe LuckyMigrator::Migration::V1 do
     end
     exists.should be_false
   end
+
+  pending "statement execution order" do
+    Spec.after_each do
+      LuckyRecord::Repo.db.exec "DROP TABLE execution_order;"
+    end
+
+    it "runs execute statements in the order they were called" do
+      MigrationWithOrderDependentExecute::V998.new.up
+      columns = LuckyRecord::Repo.table_columns("execution_order")
```

Comment:
Feel free to update the dependency in this PR

Reply:
Sorry I was mistaken. The `LuckyRecord::Repo.table_columns` method isn't in the latest release of `LuckyRecord`

Reply:
Would it make sense to run some manual SQL to get the column names?

Reply:
Yeah definitely. I added a method in the spec.

I also added arguments to `Migration#up` and `Migration#down` to disable logging so its quiet during the spec.